planning/motion_velocity_planner/autoware_motion_velocity_planner_node/src/node.cpp

```diff
@@ -151,45 +123,59 @@ void MotionVelocityPlannerNode::on_unload_plugin(
 }
 
 // NOTE: argument planner_data must not be referenced for multithreading
-bool MotionVelocityPlannerNode::is_data_ready() const
+bool MotionVelocityPlannerNode::update_planner_data()
 {
-  const auto & d = planner_data_;
   auto clock = *get_clock();
-  const auto check_with_msg = [&](const auto ptr, const auto & msg) {
+  auto is_ready = true;
+  const auto check_with_log = [&](const auto ptr, const auto & log) {
     constexpr auto throttle_duration = 3000;  // [ms]
     if (!ptr) {
-      RCLCPP_INFO_THROTTLE(get_logger(), clock, throttle_duration, msg);
+      RCLCPP_INFO_THROTTLE(get_logger(), clock, throttle_duration, log);
       return false;
+      is_ready = false;
```

Comment:
shouldn't this be before the return statement?

Reply:
You are right, thank you for spotting the mistake.
Fixed in f96271c61669f21a0b290b169add67ccabd01ebf